mmv1/products/compute/NodeTemplate.yaml

```diff
@@ -175,3 +179,23 @@ properties:
     enum_values:
       - 'ENABLED'
       - 'NONE'
+  - name: 'disks'
+    type: Array
+    description: |
+      List of the type, size and count of disks attached to the
+      node template
+    item_type:
+      type: NestedObject
+      properties:
+        - name: 'diskCount'
+          type: Integer
+          description: |
+            The number of disks exposed to this node template.
+        - name: 'diskType'
+          type: String
+          description: |
+            The type of disks exposed to this node template.
```

Comment:
Can you copy the description of this field from the [API doc](https://cloud.google.com/compute/docs/reference/rest/v1/nodeTemplates) to here?

Reply:
I've made all three match the API field descriptions.